### PR TITLE
Update sidecar AWS verification

### PR DESCRIPTION
**Pull request by jsenon**
Add verification without ssh access on master node. Perform check directly with kubectl client.

### CODE REVIEW

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 1]
```diff
@@ -208,12 +208,23 @@ kops rolling-update cluster
...
+
+With multiple Masters:
 ```bash
-/bin/sh -c /usr/local/bin/kube-apiserver --address=127.0.0.1 --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Initializers,NodeRestriction,Priority [...]
+for i in `kubectl get pods -nkube-system | grep api | awk '{print $1}'` ; do  kubectl describe pods -nkube-system $i | grep "/usr/local/bin/kube-apiserver"  ; done
+```
+
+Ouput should be:
+```bash
+[...] --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Initializers,NodeRestriction,Priority [...]
```
Comment (ayj) [inline _docs/setup/kubernetes/sidecar-injection.md]: nit: we can remove `Initializers` from the required admission controllers list (line 192 above).

Comment (jsenon) [reply _docs/setup/kubernetes/sidecar-injection.md]: Ok, I reuse existing one created by Kops. I will update.

**Comment by jsenon** [comment]
Could we merge this PR?

**Comment by ayj** [comment]
```
- ./_rakesite/test/docs/setup/kubernetes/sidecar-injection.html
  *  External link https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster failed: got a time out (response code 301)
```

`ci/circleci` presubmit is failing. Looks like you need to add a trailing `/` to one of your links (see above)

**Comment by jsenon** [comment]
could you check again ? url is available and other links doesn't contains `/`

**Comment by ayj** [comment]
```
$ curl -s -I https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster |grep -e ^HTTP -e ^location
HTTP/2 301 
location: /docs/tasks/tls/managing-tls-in-a-cluster/
```

The non-slash URL is available but redirects. And our link checker seems to be enforcing use of non-redirect links. cc @ldemailly

**Comment by ldemailly** [comment]
yes our link checker correctly enforces that one shouldn't need 2 rountrips to start serving a page

**Comment by jsenon** [comment]
Update done

**Pull request merged**